features/projects/components/project-list/project-list.module.scss

```diff
@@ -15,3 +15,36 @@
     grid-template-columns: repeat(auto-fit, 400px);
   }
 }
+
+.loaderwrapper {
+  display: flex;
+  justify-content: center; /* Center horizontally */
+  align-items: center; /* Center vertically */
+  height: 100vh; /* Full height to center it vertically in the viewport */
+  // margin-top: 0px;
+
+  @media (min-width: breakpoint.$desktop) {
+    margin-top: -300px;
+  }
+}
+
+.loader {
+  border: 8px solid #f9f5ff; /* Light grey */
+  border-top: 8px solid #7f56d9; /* Blue */
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
```

Comment:
Here you could use the global SCSS variables to be consistent with the rest of the codebase